modules/ExperimentBrowser/src/server/routes/ExperimentBrowserRoutes.coffee

```diff
@@ -5,8 +5,12 @@ Add this line to public/src/modules/ModuleMenus/src/client/ModuleMenusConfigurat
 
 ###
 
+path = require 'path'
```

Comment:
Does this get used? If not, should remove it. If it is getting used, it'd be better to move it into the route that's using it.

Reply:
Removed it!